.github/workflows/test-bacon.yml

```diff
@@ -0,0 +1,30 @@
+name: Test Bacon QR Code Provider
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        php-version: ['5.6', '7.0', '7.1', '7.2', '7.3', '7.4', '8.0', '8.1']
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - uses: shivammathur/setup-php@v2
+      with:
+        php-version: ${{ matrix.php-version }}
+        tools: composer
+        coverage: xdebug
+        ini-values: error_reporting=E_ALL
+
+    - uses: ramsey/composer-install@v1
+
+    - run: composer require bacon/bacon-qr-code
```

Comment:
Why not just make it a dev dependency? Same with the other third-party libraries? They're necessary to run the full test suite, and would avoid having an independent workflow across all supported PHP versions for this one (or several) external dependencies?

Reply:
in reality, the end user would only install bacon or endroid so the tests should recreate that environment

also adding them both as development dependencies could get complicated as endroid relies on bacon